testing/jormungandr-testing-utils/src/testing/jormungandr/process.rs

```diff
@@ -72,10 +77,12 @@ impl JormungandrProcess {
         alias: String,
     ) -> Result<Self, StartupError> {
         let stdout = child.stdout.take().unwrap();
+        let stderr = RefCell::new(OutputCollector::new(child.stderr.take().unwrap()));
 
         Ok(JormungandrProcess {
             child,
             temp_dir,
+            stderr,
             alias,
             grpc_client: JormungandrClient::new(node_config.p2p_listen_address()),
             logger: JormungandrLogger::new(stdout),
```

Comment:
Maybe, is it better to put `stderr` into the `JormungandrLogger` struct, so it will maintain any logging and messaging which is coming from the node ? What do you think ?

Reply:
I thought about this at the beginning and then decided to go for adding it to `JormungandrProcess` because those are at a slightly different level but I agree with your point that it is probably simpler